xml_converter/src/xml_converter.cpp

```diff
@@ -61,30 +67,181 @@ void write_xml_file(string xml_filepath, map<string, Category>* marker_categorie
     outfile.close();
 }
 
-void write_protobuf_file(string proto_filepath, map<string, Category>* marker_categories, vector<Parseable*>* parsed_pois) {
-    ofstream outfile;
-    waypoint::Waypoint proto_message;
+void create_proto_categories(waypoint::Category proto_category, set<string>* proto_categories) {
```

Comment:
I think there are some diffs in this file that are actually a part of #114 so make sure that that branch is synced up with this one. Though maybe wait until I finish reviewing that branch, hopefully soon, so that you only have to do that comparison once instead of every time there is a revision.

Reply:
The changed from that PR have been merged in.